### ISSUE TITLE
♿ 🐞 Image elements have [alt] attributes that contain redundant text in the Post Info widget

### ISSUE THREAD

**Issue by jlemonz** [elementor/elementor#25440]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Hi, I use the post info widget with an author element and I select avatar in a loop item, but this gives error: 

 "Image elements have [alt] attributes that contain redundant text."   In lighthouse. 

I tried changing alt text etc but nothing worked. 

### Steps to reproduce

1 create loop item
2 use post widget with author with avatar
3 you can use gravatar or custom upload
4 check lighthouse

### Expected behavior

no lighthouse errors

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: nginx/1.25.2
	MySQL version: mariadb.org binary distribution v10.11.5-MariaDB-1:10.11.5+maria~ubu2004
	PHP Version: 8.2.7
	PHP Memory Limit: 256M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.3
	Site URL: https://smoothiesenzo.nl
	Home URL: https://smoothiesenzo.nl
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /blog/%postname%/
	Language: nl_NL
	Timezone: 1
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.0.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: nl-NL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.2.7
		Author: WP Engine

	Advanced Database Cleaner
		Version: 3.1.4
		Author: Younes JFR.

	Anti-Spam by CleanTalk
		Version: 6.29
		Author: СleanTalk - Anti-Spam Protection <welcome@cleantalk.org>

	Autoptimize
		Version: 3.1.11
		Author: Frank Goossens (futtta)

	Complianz | GDPR/CCPA Cookie Consent
		Version: 7.0.4
		Author: Really Simple Plugins

	Confetti
		Version: 1.3.6
		Author: WP Sunshine

	Elementor
		Version: 3.20.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.20.0
		Author: Elementor.com

	Email Log
		Version: 2.4.9
		Author: Sudar

	MailerSend - Official SMTP Integration
		Version: 1.0.3
		Author: MailerSend

	Performance Lab
		Version: 2.9.0
		Author: WordPress Performance Team

	ProfilePress
		Version: 4.15.3
		Author: ProfilePress Membership Team

	Rank Math SEO
		Version: 1.0.214
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.57
		Author: Rank Math

	Safe SVG
		Version: 2.2.2
		Author: 10up

	ShortPixel Image Optimizer
		Version: 5.5.5
		Author: ShortPixel - Convert WebP/AVIF & Optimize Images

	Site Kit by Google
		Version: 1.122.0
		Author: Google

	WPCode Lite
		Version: 2.1.9
		Author: WPCode


== Must-Use Plugins ==
	Kinsta Must-use Plugins
		Version: 3.0.1
		Author: Kinsta Team


== Eigenschappen ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	facebook_sdk: Active


== Elementor experimenten ==
	Verbeterd laden van assets: Actief
	Verbeterd laden van CSS: Actief
	Pictogrammen inline lettertype: Actief
	Extra aangepaste breakpoints: Actief
	admin_menu_rearrangement: Standaard inactief
	Flexbox container: Actief
	Upgrade Swiper bibliotheek: Actief
	Raster container: Actief
	Default to New Theme Builder: Standaard actief
	Hello Theme header & footer: Standaard inactief
	Editor top balk: Actief
	Geoptimaliseerd laden van Gutenberg: Actief
	Bouwen met AI: Standaard actief
	Landingspagina's: Standaard actief
	Geneste elementen: Actief
	Lazy-load achtergrondafbeeldingen: Actief
	Afbeelding laden optimaliseren: Actief
	Pagina's paneel: Standaard inactief
	Notes: Standaard actief
	Display Conditions: Actief
	Form Submissions: Standaard actief
	Menu: Actief
	Taxonomy Filter: Actief


== Log ==
	
Log: showing 20 of 342024-03-11 15:03:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.20.0-beta3',
  'to' => '3.20.0',
)]
2024-03-11 15:03:11 [info] elementor-pro::elementor_pro_updater Started 
2024-03-11 15:03:11 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-03-11 15:03:11 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-03-11 15:03:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.20.0-beta3',
  'to' => '3.20.0',
)]
2024-03-11 15:03:11 [info] elementor::elementor_updater Started 
2024-03-11 15:03:11 [info] Elementor/Upgrades - _on_each_version Start  
2024-03-11 15:03:11 [info] Elementor/Upgrades - _on_each_version Finished 
2024-03-11 15:03:12 [info] Elementor/Upgrades - _on_each_version Finished 
2024-03-11 15:03:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0-beta3',
  'to' => '3.20.0-beta4',
)]
2024-03-11 15:03:12 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0-beta3',
  'to' => '3.20.0-beta4',
)]
2024-03-11 15:06:10 [info] elementor::elementor_updater Started 
2024-03-11 15:06:10 [info] Elementor/Upgrades - _on_each_version Start  
2024-03-11 15:06:10 [info] Elementor/Upgrades - _on_each_version Finished 
2024-03-11 15:06:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0-beta4',
  'to' => '3.20.0',
)]
2024-03-14 20:29:33 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0',
  'to' => '3.20.1',
)]
2024-03-14 20:29:33 [info] elementor::elementor_updater Started 
2024-03-14 20:29:33 [info] Elementor/Upgrades - _on_each_version Start  
2024-03-14 20:29:33 [info] Elementor/Upgrades - _on_each_version Finished 
2024-03-14 20:29:33 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.20.0',
  'to' => '3.20.1',
)]

PHP: showing 5 of 5PHP: 2024-03-01 14:39:08 [notice X 71][../wp-content/plugins/elementor/includes/base/controls-stack.php::228] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-03-01 14:43:39 [warning X 1][../wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-03-04 18:42:36 [warning X 6][../wp-content/plugins/elementor/core/page-assets/data-managers/font-icon-svg/font-awesome.php::50] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-03-11 15:03:11 [warning X 1][../wp-content/plugins/elementor/core/files/manager.php::111] unlink(../wp-content/uploads/elementor/css/post-9037.css): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-03-11 15:03:12 [warning X 1][../wp-content/plugins/elementor/core/files/uploads-manager.php::699] rmdir(../wp-content/uploads/elementor/tmp/): No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 3 of 3JS: 2024-03-01 13:45:48 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.19.4:3:647925] Cannot read properties of undefined (reading 'isDesignable') 
JS: 2024-03-06 17:50:29 [error X 6][../wp-content/plugins/elementor-pro/assets/js/elements-handlers.min.js?ver=3.20.0-beta3:2:17405] o.trigger is not a function 
JS: 2024-03-14 19:32:20 [error X 1][../wp-content/plugins/elementor/assets/js/packages/editor-documents/editor-documents.min.js?ver=3.20.1:1:1686] Cannot read properties of null (reading 'config') 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibiliteit niet gespecificeerd

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @jlemonz! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Also, keep in mind that both Page Speed Insights and Lighthouse reports are only a set of suggestions or highlights of opportunities to improve some aspects of the pages. It doesn't mean they point an error or a bug. Sometimes some suggestions will contradict other suggestions. In this case, if alt tags are removed, there is also a warning.

We will pass this to our accessibility experts and will let you know if this is indeed a bug, or simply a design choice. Thank you for understanding.

Kind regards

**Comment by nicholaszein** [comment]
Hello there @jlemonz!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.21.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
